ecl/hqlcpp/hqlres.cpp

```diff
@@ -479,7 +479,8 @@ bool ResourceManager::flush(StringBuffer &filename, const char *basename, bool f
         VStringBuffer label("%s_%u_txt_start", type, id);
         if (generateClang)
         {
-            fprintf(f, " .section __TEXT,%s_%u\n", type, id);
+            if (id == 1000)  // Can't afford a section per resource, but these ones need it as they are retrieved without loading the dll
```

Comment:
@afishbeck @richardkchapman 
Is this valid.  For instance IN wuwebview:

if (dll->getResource(len, data, "RESULT_XSD", (unsigned) id) && len>0)

and it looks like there are calls to get resources from CWsEclBinding::onGet()

Reply:
I think that comes from a loaded dll (though I may be wrong)

Reply:
Actually looks like it might not - it's from a ILoadedDllEntry but that may have been created via loadDllResources, which memory-maps the dll rather than loading it.

In which case I suspect I will need to change this fix to restrict the number of sections or 250 or so, and make it OSX specific. Anyone trying to view the 251'st result from a workunit when running their eclwatch on OSX may get an error, but I don't expect to be flooded by bug reports of that.